test-data/unit/check-attr.test

```diff
@@ -532,6 +532,21 @@ class C:
 
 [builtins fixtures/list.pyi]
 
+[case testAttrsUsingUnsupportedConverter]
+import attr
+class Thing:
+    def do_it(self, int) -> str:
+        ...
+thing = Thing()
+def factory(default: int):
+   ...
+@attr.s
+class C:
+    x: str = attr.ib(converter=thing.do_it)
+    y: str = attr.ib(converter=lambda x: x)
+    z: str = attr.ib(converter=factory(8))
```

Comment:
These should give errors until supported.

By the way I am thinking about how to support this. One possible way is to just accept type checker at a later stage. This would require a second part of the plugin that would be triggered on visiting this `TypeInfo` in type checker. We also need to be sure to mark all nodes that need this type as deferred. TBH I am sure good support for converters worth all this (including potential performance problems). So maybe we can just propose a workaround like always explicitly define your converters i.e. use
```python
def str_first_item(seq: List[int]) -> int:
    """A docstring for why one needs this"""
    return str(seq[0])

@attr.s
class C:
    x: str = attr.ib(converter=str_first_item)
```
instead of
```python
attr.s
class C:
    x: str = attr.ib(converter=lambda x: str(x[0]))
```

Reply:
Yeah. That's what I was already thinking for my own code.  Should I mention this in the error message?  "only functions supported" or something like that?

Reply:
> Should I mention this in the error message? "only functions supported" or something like that?

Yes, this will be helpful. Maybe like "only named functions" (to make it clear that lambdas are not fully supported).